lib/flask_tasks.py

```diff
@@ -15,13 +16,20 @@
 from flask import url_for as _url_for, _request_ctx_stack
 from werkzeug.exceptions import HTTPException, InternalServerError
 
+import threaded_logging
+
 logging.basicConfig(format='%(asctime)s | %(name)s | %(message)s',
                     level=logging.DEBUG)
 
 
 tasks_bp = Blueprint('tasks', __name__)
 tasks = {}
 testing = False
+# in sec
```

Comment:
instead of comments it is better to rename next variables with _sec_ or _seconds_ suffix, isn't it?

Reply:
done